src/main/java/com/autotune/experimentManager/handler/MetricCollectionHandler.java

```diff
@@ -160,9 +162,10 @@ public void execute(ExperimentTrial experimentTrial, TrialDetails trialDetails,
                                 }
                                 if (null != updatedContainerQuery) {
                                     LOGGER.debug("Updated Query - " + updatedContainerQuery);
+                                    AuthenticationConfig noAuthConfig = AuthenticationConfig.noAuth();
                                     String queryResult = (String) ado.getValueForQuery(experimentTrial.getDatasourceInfoHashMap()
                                             .get(containerMetric.getDatasource())
-                                            .getUrl().toString(), updatedContainerQuery);
+                                            .getUrl().toString(), updatedContainerQuery, noAuthConfig);
```

Comment:
authoconfig should get tagged to datasource obj and 
datasource obj should get tagged to Experiment

so data pass using parameter should be just experiment Obj everywhere